libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/model/roadobject/reststop/RestStopType.kt

```diff
@@ -10,15 +12,26 @@ object RestStopType {
      * This means that the type is either erroneous,
      * or there's a new type available server-side which will be exposed in the SDK in the future.
      */
-    const val Unknown = -1
+    const val UNKNOWN = -1
 
     /**
      * Describes a rest area.
      */
-    const val RestArea = 1
+    const val REST_AREA = 1
 
     /**
      * Describes a service area.
      */
-    const val ServiceArea = 2
+    const val SERVICE_AREA = 2
+
+    /**
+     * Rest stop type.
+     */
+    @Retention(AnnotationRetention.SOURCE)
```

Comment:
```suggestion
    @Retention(AnnotationRetention.RUNTIME)
```

Reply:
We use `@Retention(AnnotationRetention.SOURCE)` across our codebase. Should we change all of them? 🤔

Reply:
@RingerJK  Going to go ahead and merge here. We can discuss further, could you cut a ticket when you have a chance? 🙏

Reply:
> We use `@Retention(AnnotationRetention.SOURCE)` across our codebase. Should we change all of them? 🤔

yeah, definitely for public API

Reply:
following up at https://github.com/mapbox/mapbox-navigation-android/issues/4145